src/controller/python/ADVANCED_USAGE.md

```diff
@@ -0,0 +1,234 @@
+# Python CHIP Device Controller - Advanced usage
+
+## BLE Virtualization on Linux (Optional)
+
+If you would like to setup virtual BLE central and peripheral in the same
+machine, then follow below steps:
+
+```
+cd third_party/bluez/repo
+./bootstrap
+third_party/bluez/repo/configure --prefix=/usr --mandir=/usr/share/man --sysconfdir=/etc --localstatedir=/var --enable-experimental --with-systemdsystemunitdir=/lib/systemd/system --with-systemduserunitdir=/usr/lib/systemd --enable-deprecated --enable-testing --enable-tools
+make
+```
+
+Note: You also need to install several packages on RPi if you want to build
+bluez
+
+```
+sudo apt-get install libtool m4 automake autotools-dev libudev-dev libical-dev libreadline-dev
+```
+
+Run bluetoothd:
+
+```
+sudo third_party/bluez/repo/src/bluetoothd --experimental --debug &
+```
+
+Bring up two virtual ble interface:
+
+```
+sudo third_party/bluez/repo/emulator/btvirt -L -l2
+```
+
+You can find the virtual interface by `hciconfig` command:
```

Comment:
For me we are missing some verb here. Like: "by running/calling/invoking/typing command".